src/main/java/net/sf/jabref/logic/cleanup/CleanupPreset.java

```diff
@@ -0,0 +1,210 @@
+/*  Copyright (C) 2003-2015 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+package net.sf.jabref.logic.cleanup;
+
+public class CleanupPreset {
```

Comment:
Hehe, actually the whole point of the PR was that I felt the pain :). I wanted to implement cleanup actions for onSave but then quickly realized that I would replicate a lot of code. Thus first this refactoring. 

In some way or another the cleanup process has to know a bunch of boolean values and so I figured it would be best to encapsulate them in a simple data class.